drawCommand.js

```diff
@@ -0,0 +1,126 @@
+import db from "./database.js";
```

Comment:
similarly, by convention, you want to import with the same name as the filename (usually), so in this case it woudl be `import database from "./database.js"`